src/main/java/org/jabref/gui/groups/GroupDialog.java

```diff
@@ -610,7 +615,30 @@ private void openBrowseDialog() {
             .addExtensionFilter(StandardFileType.AUX)
             .withDefaultExtension(StandardFileType.AUX)
             .withInitialDirectory(Globals.prefs.get(JabRefPreferences.WORKING_DIRECTORY)).build();
-        dialogService.showFileOpenDialog(fileDialogConfiguration).ifPresent(file -> texGroupFilePath.setText(file.toAbsolutePath().toString()));
+        dialogService.showFileOpenDialog(fileDialogConfiguration).ifPresent(file -> texGroupFilePath.setText(relativize(file.toAbsolutePath()).toString()));
+    }
+
+    private Path relativize(Path path) {
+        List<Path> fileDirectories = getFileDirectoriesAsPaths();
+        if (!path.isAbsolute()) {
+            return path;
+        }
+
+        for (Path directory : fileDirectories) {
+            if (path.startsWith(directory)) {
```

Comment:
Better use the path.toString startsWith might behave differently than expected: https://stackoverflow.com/a/30157146